doc/getting_started/index.rst

```diff
@@ -42,11 +42,14 @@ First, install the ``west`` binary and bootstrapper:
 .. code-block:: console
 
    # Linux
-   pip3 install --user west
+   pip3 install --user west~=0.5.8
```

Comment:
The meaning of this is: `>=v0.5.8` and `==0.5.*`, i.e., fetch the latest v0.5.x.